lib/snyk-to-html.js

```diff
@@ -42,14 +42,45 @@ function generateTemplate(data, template) {
   return Handlebars.compile(htmlTemplate)(data);
 }
 
+function mergeData(dataArray) {
+  const vulnsArrays = dataArray.map(d => d.vulnerabilities);
+  const aggregateVulnerabilities = [].concat.apply([], vulnsArrays);
+
+  const totalUniqueCount =
+    dataArray.reduce((acc, item) => acc + item.uniqueCount, 0);
+  function fnPathCount(obj) {
+    // sample: "85 vulnerable dependency paths"
+    return parseInt(obj.summary.match(/\d+/).pop());
```

Comment:
I fear this could fail (with `Cannot read property 'pop' of null` error) when there won't be any vulnerability. Could you please add a test which combines more vuln-free outputs?

Reply:
would `.pop || 0` handle this?

Reply:
```
    const matchResult = obj.summary.match(/\d+/);
    return parseInt(matchResult ? matchResult.pop() : '0');
```

Reply:
No need for a dedicated test, I modified the code to explicitly return 0 in the absence of a match.